src/config.ts

```diff
@@ -82,180 +84,35 @@ import { LoginOptions, DeepPartial } from './auth0-session';
-
-  /**
-   * The Client ID for your application.
-   * Can be read from CLIENT_ID instead.
-   */
-  clientID: string;
-
-  /**
-   * The Client Secret for your application.
-   * Required when requesting access tokens.
-   * Can be read from CLIENT_SECRET instead.
-   */
-  clientSecret?: string;
-
-  /**
-   * Integer value for the system clock's tolerance (leeway) in seconds for ID token verification.`
-   * Default is 60
-   */
-  clockTolerance: number;
-
-  /**
-   * Integer value for the http timeout in ms for authentication requests.
-   * Default is 5000
-   */
-  httpTimeout: number;
-
-  /**
-   * To opt-out of sending the library and node version to your authorization server
-   * via the `Auth0-Client` header. Default is `true
-   */
-  enableTelemetry: boolean;
-
-  /**
-   * @ignore
-   */
-  errorOnRequiredAuth: boolean;
-
-  /**
-   * @ignore
-   */
-  attemptSilentLogin: boolean;
-
-  /**
-   * Function that returns an object with URL-safe state values for `res.oidc.login()`.
-   * Used for passing custom state parameters to your authorization server.
-   * Can also be passed in to {@link HandleLogin}
-   *
-   * ```js
-   * {
-   *   ...
-   *   getLoginState(req, options) {
-   *     return {
-   *       returnTo: options.returnTo || req.originalUrl,
-   *       customState: 'foo'
-   *     };
-   *   }
-   * }
-   * ``
-   */
-  getLoginState: (req: IncomingMessage, options: LoginOptions) => Record<string, any>;
-
-  /**
-   * Array value of claims to remove from the ID token before storing the cookie session.
-   * Default is `['aud', 'iss', 'iat', 'exp', 'nbf', 'nonce', 'azp', 'auth_time', 's_hash', 'at_hash', 'c_hash' ]`
-   */
-  identityClaimFilter: string[];
-
-  /**
-   * Boolean value to log the user out from the identity provider on application logout. Default is `true`
-   */
-  idpLogout: boolean;
-
-  /**
-   * String value for the expected ID token algorithm. Default is 'RS256'
-   */
-  idTokenSigningAlg: string;
-
-  /**
-   * REQUIRED. The root URL for the token issuer with no trailing slash.
-   * This is `https://` plus your Auth0 domain
-   * Can use env key ISSUER_BASE_URL instead.
-   */
-  issuerBaseURL: string;
-
-  /**
-   * Set a fallback cookie with no SameSite attribute when response_mode is form_post.
-   * Default is true
-   */
-  legacySameSiteCookie: boolean;
-
-  /**
-   * @ignore
-   */
-  authRequired: boolean;
-
+export interface Config extends SessionLayerConfig {
   /**
-   * Boolean value to automatically install the login and logout routes.
+   * Configuration parameters to override the default authentication URLs.
    */
   routes: {
     /**
-     * @ignore
+     * Relative path to the login handler.
      */
-    login: string | false;
+    login: string;
+
+    /**
+     * Either a relative path to the application or a valid URI to an external domain.
+     * The user will be redirected to this after a login has been performed.
+     */
+    postLoginRedirect: string;
```

Comment:
If we're going to create a `postLoginRedirect` option, then it should work the same as `postLogoutRedirect` - in that it is the default in `auth0-session#logout` if `returnTo` is not set https://github.com/auth0/nextjs-auth0/blob/beta/src/auth0-session/handlers/logout.ts#L17

Reply:
Good catch, I'll add it.